src/jp/mcbe/fuyutsuki/Texter/task/SendTextsTask.php

```diff
@@ -34,7 +34,7 @@ public function onRun(): void {
 			$this->onSuccess();
 		}else {
 			$floatingText = array_shift($this->remain);
-			$floatingText->sendToPlayer($this->target, $this->type);
+			if ($this->target->isConnected()) $floatingText->sendToPlayer($this->target, $this->type);
```

Comment:
To avoid repeating the process in vain, cancel this task immediately if the target (player) is not connected.